gradle/libs.versions.toml

```diff
@@ -87,6 +87,7 @@ r2dbc = { module = "io.r2dbc:r2dbc-spi", version = "1.0.0.RELEASE" }
 testContainers-mysql = { module = "org.testcontainers:mysql", version.ref = "testContainers" }
 testContainers-postgres = { module = "org.testcontainers:postgresql", version.ref = "testContainers" }
 testContainers-r2dbc = { module = "org.testcontainers:r2dbc", version.ref = "testContainers" }
+jts = { module = "org.locationtech.jts:jts-core", version = "1.18.2" }
```

Comment:
This is one of the dependencies that I found that can be used to represent that Geometry object.